elyra/pipeline/processor_kfp.py

```diff
@@ -51,6 +51,8 @@ def process(self, pipeline):
         t0_all = time.time()
         timestamp = datetime.now().strftime("%m%d%H%M%S")
         pipeline_name = f'{pipeline.name}-{timestamp}'
```

Comment:
Shouldn't `pipeline_name` be `pipeline.name` (sans timestamp)?

Reply:
Yes. I didn't realize that this implementation violates the definition in https://github.com/elyra-ai/elyra/pull/1284#issuecomment-777779353, which states that the value is the pipeline file name or the value the user provided when the run was initiated.

However, the issue is that that value currently does not surface in the KFP UI, as shown in the screen captures for this scenario: https://github.com/elyra-ai/elyra/pull/1284#issuecomment-778510188.

Reply:
fixed!